### ISSUE TITLE
Request: Fontsize

### ISSUE THREAD

**Issue by glucee** [geeeeeeeeek/electronic-wechat#528]
Could you add a feature to change the size of font ? The font is really small in my screen, thanks.

#### Specifications

- Version of Electron (run `$ electron --version`): `v`
- OS: `Ubuntu 16.04` 
- Stack trace from the error message (if any)

```
<Stack trace here>
```

**Comment by Roller44** [comment]
Is it any chance that because the resolution of your screen is high? Do 
you think the font is small while using browser?

**Comment by glucee** [comment]
yes, the font in browser is small as well but i could scale it up by zooming in. Our wechat doesn't support that.
